Library/Formula/ale.rb

```diff
@@ -0,0 +1,17 @@
+class Ale < Formula
+  homepage "https://darker0n.github.io/ale"
+  url "https://github.com/darker0n/ale/archive/v0.1.1.tar.gz"
+  version "0.1.1"
+  sha256 "2bae9392f02dc6fc1a77c637fa0139a26648b0303a63eae9b46661999985c679"
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec
```

Comment:
` ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages" ` (Presuming it needs 2.7, of course).